src/app/components/champion/ChampionView.tsx

```diff
@@ -28,34 +29,55 @@ export const ChampionView: FC<ChampionProps> = ({champion, callback, championsLi
     const useStyles = makeStyles((theme: Theme) => ({
         icon: {
             verticalAlign: 'bottom',
-            height: 25,
-            width: 25,
+            height: 30,
+            width: 30,
+            color: theme.palette.secondary.main
         },
-        slider: {
-            paddingTop: '14%'
-        },
-        statBox: {
-            backgroundColor: theme.palette.primary.main
+        componentBox: {
+            backgroundColor: theme.palette.primary.main,
+            marginTop: '35px'
         },
         avatarSearch: {
-            height: '22%',
-            width: '22%',
-            marginRight:'5%'
-        }
+            height: '50px',
+            width: '50px',
+            marginRight: '5%'
+        },
+        avatarDisplay: {
+            height: '75px',
```

Comment:
With height like those, I truly hope you developped mobile first

Reply:
I ran into some trouble while using % for one of the icons, and kept using px as a result on this commit.